tests/test.py

```diff
@@ -1036,6 +1041,37 @@ def fork_connector(self, connector_port: int, acraserver_port: int,
         logging.info("fork connector finished [pid={}]".format(process.pid))
         return process
 
+    def fork_ocsp_server(self, port: int, check_connection: bool=True):
+        logging.info("fork OpenSSL OCSP server with port {}".format(port))
+
+        ocsp_server_connection = self.get_ocsp_server_connection_string(port)
+
+        args = {
+            'port': port,
+            'index': TEST_TLS_OCSP_INDEX,
+            'rsigner': TEST_TLS_OCSP_CERT,
+            'rkey': TEST_TLS_OCSP_KEY,
+            'CA': TEST_TLS_CA,
+        }
+
+        cli_args = sorted(['-{}={}'.format(k, v) for k, v in args.items()])
+        print('openssl ocsp args: {}'.format(' '.join(cli_args)))
```

Comment:
better to use `logging` module instead print. it will allow us to change logs level and filter some levels if need.

Reply:
ping